Signal/src/Models/TSMessageAdapaters/TSPhotoAdapter.m

```diff
@@ -75,8 +75,7 @@ - (UIView *)mediaView {
         UIImage *image = self.attachment.image;
         if (!image) {
             DDLogError(@"%@ Could not load image: %@", [self tag], [self.attachment mediaURL]);
-            OWSAssert(0);
```

Comment:
If the image or video is corrupt or invalid.  That should be rare but possible in production.  I removed this debug assert since the attachments created by debug UI's "create tiny attachments are all invalid.  I feel like its worth it - this assert was buying us very little.